source/SavedGame.cpp

```diff
@@ -65,6 +65,21 @@ void SavedGame::Load(const string &path)
 					shipName = child.Token(1);
 				else if(child.Token(0) == "sprite" && child.Size() >= 2)
 					shipSprite = SpriteSet::Get(child.Token(1));
+				// If the selected ship is parked or in another system, it cannot be the active
+				// flagship, so clear the sprite and continue the loop until it is found
```

Comment:
```suggestion
				// flagship, so clear the sprite and continue the loop until it is found.
```
Could the first ship in the list potentially be in system but disabled, therefore meaning it can't be the flagship?

Reply:
Ah, good catch. Since all the code to check if a ship is disabled is private in the Ship class, I'm adding a tag for Ship::Write() which'll write out `disabled`. This won't get read when loading a full save, since the game will detect that from the full attributes, but it'll quickly let it find out when doing the quick read.